### PR TITLE
Rats, we're rats, we're the rats: Rhodian (Rodeetch) race update

### PR DESCRIPTION
**Pull request by Kepteyn**
## About The Pull Request

Adds ~~rhodians~~ rodeetch; a race of ratmen who, due to the maddenning influence of the inhumen gods, who they had forsaken, destroyed the city of gold - a rich kingdom build in worship of Astrata - in a catacysm called 'The Final Feast.' Split between those who follow the godess of the sun and those who follow the inhumen gods.

~~Rhodians~~ Rodeetch use the standard humanoid body, with the anthromorph rodent features and markings as required options when you make them. They are classed as a tolerated species, and are eligible to be nobles.

The racial stats are:
male: strength = -1, perception = 1, intelligence = -1, constitution = 0, endurance = 0, speed = 2, fortune = -1
female: strength = -2, perception = 1, intelligence = 0, constitution = -1, endurance = 0, speed = 3, fortune = -1

## Why It's Good For The Game

Rhodians (now renamed to rodeetch per Salty Rat's request) are listed as a species in the lore primer, but up until now have not have a species in-game to represent them. This adds them in, at last. Now go play-kill with them, yes-yes!

rhodian

### CODE REVIEW

**Comment by Kepteyn** [comment]
Can't call it Ratwood Keep without the rats

**Comment by pismorizzo** [comment]
A worthy Inhumane god addition.

**Comment by Hmeister-real** [comment]
remove the -1 fortune, or add -1 f to female too

**Comment by DeerJesus** [comment]
Add mouse ears as well - not murid, already got that - to rhodian.

**Comment by Kepteyn** [comment]
> Codewise I made my review Balancewise and Lorewise (whether the god should exist, what is the Rhodian lore etc.) Useroth and other staff can weigh in

Thank you greatly for your help, the changes have been made.


> remove the -1 fortune, or add -1 f to female too

I was mistaken and thought that the female stats were modified ontop of the male ones. This has been fixed.

File: code/modules/mob/living/carbon/human/species_types/furry/ratmen.dm [hunk 1]
```diff
@@ -0,0 +1,176 @@
+/mob/living/carbon/human/species/ratpeople
+	race = /datum/species/ratpeople
+
+/datum/species/ratpeople
+	name = "Rodeetch" 
+	id = "ratpeople"
+	desc = "<b>Rhodian</b><br>\
+	The Rodent-Like Rodeetch are a race that had faced the rise and fall of their grand capital and survived with a distinct and deep religious divide. \
```
[the remark this answers is not in the document]
Comment (Kepteyn) [reply code/modules/mob/living/carbon/human/species_types/furry/ratmen.dm:8]: Rodeetch is the new name of the Rhodians. I'll correct the description, missed the name there.

**Pull request closed without merging**